tests/modules/nf-core/eklipse/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { EKLIPSE } from '../../../../modules/nf-core/eklipse/main.nf'
+
+workflow test_eklipse {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        '/usr/local/bin/data/test_illumina.bam'
```

Comment:
Can you use a file from the test data repository instead please? This way the tests will also work in the CI

Reply:
Hi,thanks for the review :)  unfortunately the data in the tests does not work, it needs more data than what is given in the mitochondria bam file present in the test_data, that is why I took the one present inside the environment. The tests pass locally but fail in GitHub, is it due to the files not being in test_data?

Reply:
@nvnieuwk do you have any suggestions on how to address this?

Reply:
It could be indeed, how big is the file in the container? Maybe you can add it to the repository and try again?

Reply:
My guess is that nextflow can't find the file in the input (`path()`) of the module

Reply:
There are 3 files, the reference 70kb, the bam 3,5MB and bam.bai 128B
Is the bam might too big to put in test_data, or is it ok?

Reply:
I will do that then, thanks for your help!